packages/api/src/schema/schema.yaml

```diff
@@ -1109,6 +1110,24 @@ components:
                   type: string
                   description: S3 object key of the uploaded asset
                   example: ABC123/filename.mp4
+            export:
+              type: object
+              additionalProperties: false
+              description: Parameters for the export task
+              properties:
+                url:
+                  type: string
+                  description: s3 presigned url of the asset to export
+                  example: s3+https://s3.amazonaws.com/livepeer-test/ABC123/filename.mp4
+                ipfs:
+                  type: boolean
```

Comment:
This **must** be an object, not a boolean, since it will later contain sub-fields. Like for example custom service, ERC1155 metadata, etc.